apiary.apib

```diff
@@ -1476,7 +1481,12 @@ For example you should create a group for the Introduction to Programming course
                         "batch_id": "271"
                     },
                     "learning_outcomes": ["Reading Books", "Writing Essays"],
-                    "generate_private_credential": false
+                    "generate_private_credential": false,
+                    "auto_expiry": 2024,
```

Comment:
I find this misleading. "auto_expiry" should not be `2024` but `1` or `2` as in 1 year later or 2 years later.